scripts/compile-themes.js

```diff
@@ -203,5 +193,9 @@ if (require.main === module) {
     process.exit(1);
   }
 
-  compileScssFiles(path.join('src', 'theme_*.scss'), 'dist', ouiPackageName);
+  compileScssFiles(path.join('src', 'theme_*.scss'), 'dist', ouiPackageName)
+      .catch((err) => {
+        console.error(err);
+        process.exit(2);
+      });
```

Comment:
Thinking about it again, this function _shouldn't_ ever throw an error, I think. Everything that might is wrapped in a try-catch

Reply:
There is a re-throw in `compileScssFiles`.